sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraIO.java

```diff
@@ -711,6 +723,15 @@ public T getCurrent() throws NoSuchElementException {
       public CassandraIO.CassandraSource<T> getCurrentSource() {
         return source;
       }
+
+      private Mapper<T> getMapper(Session session, Class<T> enitity) {
```

Comment:
> s/enitity/entity
Fixed.

Reply:
> Actually this method can be removed (or at least inlined in their use) since there will always be a valid `mapperFactory`

See main comment below.